padertorch/ops/losses/regression.py

```diff
@@ -49,45 +49,59 @@ def log_mse_loss(estimate: torch.Tensor, target: torch.Tensor,
             TasNet: A Dissecting Approach.” ArXiv:1911.08895
             [Cs, Eess], November 20, 2019.
             http://arxiv.org/abs/1911.08895.
+
+    >>> estimate = [[1., 2, 3], [4, 5, 6]]
+    >>> target = [[2., 3, 4], [4, 0, 6]]
+    >>> log_mse_loss(torch.tensor(estimate), torch.tensor(target))
+    tensor(0.9208)
+    >>> log_mse_loss(torch.tensor(estimate), torch.tensor(target), reduction=None)
+    tensor([0.0000, 0.9208])
     """
     # Use the PyTorch implementation for MSE, should be the fastest
     return _reduce(
-        F.mse_loss(estimate, target, reduce='none').mean(dim=-1).log10(),
-        reduce=reduce
+        F.mse_loss(estimate, target, reduction='none').mean(dim=-1).log10(),
+        reduction=reduction
     )
 
 
-def sdr_loss(estimates: torch.Tensor, targets: torch.Tensor,
-             reduce: str = 'mean'):
+def sdr_loss(estimate: torch.Tensor, target: torch.Tensor,
+             reduction: str = 'mean'):
```

Comment:
How about naming them `input` and `target`, similar to many PyTorch loss functions? Also for the other loss functions here

Reply:
I am not sure, if `input` is a good name. The name hides a built-in.

Reply:
I agree, I don't like the name either. That's the reason why it's not called input already. But I thought it might be good to use the same names as PyTorch.

Reply:
I think it is more important to have the same order than the same name, so I would keep the names as they are.